Assets/MRTK/Providers/XRSDK/Controllers/GenericXRSDKController.cs

```diff
@@ -263,29 +271,7 @@ protected virtual void UpdateSingleAxisData(MixedRealityInteractionMapping inter
                     {
                         CoreServices.InputSystem?.RaiseOnInputUp(InputSource, ControllerHandedness, interactionMapping.MixedRealityInputAction);
                     }
-                }
-
-                switch (interactionMapping.InputType)
-                {
-                    case DeviceInputType.Trigger:
```

Comment:
These `DeviceInputType`s are not the same as the ones they were combined with above (Trigger and Grip vs TriggerPress and GripPress)